src/main/java/org/opensrp/domain/CustomPhysicalLocation.java

```diff
@@ -0,0 +1,19 @@
+package org.opensrp.domain;
+
+import java.util.Map;
+
+public class CustomPhysicalLocation extends PhysicalLocation {
```

Comment:
There is custom properties map in `PhysicalLocation` class, why not use that object. This will allow for consistency